modules/ROOT/pages/configuration/files/external_storage/auth_mechanisms.adoc

```diff
@@ -36,7 +33,7 @@ credentials, and background file scanning does not work.
 
 [NOTE]
```

Comment:
Question:

This writes ``[NOTE]``

while in https://github.com/owncloud/docs/edit/master/modules/ROOT/pages/configuration/server/occ_command.adoc#L2852
we write ``Note :``
and in 
https://github.com/owncloud/docs/edit/master/modules/ROOT/pages/configuration/server/occ_app_command.adoc#L656
we write ``NOTE:``

Which one should we take?

Reply:
This is an example of [an admonition block](https://asciidoctor.org/docs/asciidoc-syntax-quick-reference/#admon-bl), which can contain more complicated/sophisticated content.